bulkRoleManager/bulkRoleManager.py

```diff
@@ -773,6 +773,9 @@ def get_player_nickname(self, user: discord.Member):
             else:
                 currentNickname = array[0]
             return currentNickname
+        elif user.global_name:
```

Comment:
`global_name` isn't an attribute of `discord.Member`.

Reply:
https://discordpy.readthedocs.io/en/stable/api.html#discord.Member.global_name :( Straight from the documentation. But requires Discord API `2.3`